### ISSUE TITLE
New icon for Bookmarks and Tracks dialog

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#6561]
The current icon on Android is a star, on iOS it is "stars with lines". The issue is that we don't draw/display any stars anywhere. Our bookmarks are pins, not stars.
The original idea was to make it easier for users from Google Maps to migrate and associate it with bookmarks. But since those days Google started using many other symbols for bookmarks, not just stars. And we added imported tracks support. And plan to add tracks saving and recording.

There were already cases when users could not discover where bookmarks or tracks were stored.

In short: a new icon should make it clear. Ideally, pins with tracks. Maybe some better ideas.

**Comment by Jean-BaptisteC** [comment]
Google Maps and Maps.Me use this icon https://fonts.google.com/icons?selected=Material%20Symbols%20Outlined%3Abookmark%3AFILL%400%3Bwght%40400%3BGRAD%400%3Bopsz%4024
And other apps like Apple Plans and OSmAnd use stars icon.

**Comment by biodranik** [comment]
While Google's icon is familiar to users (it is also used in Slack as "Save for later"), it does not directly correlate to map pin symbols.

In our case, we also need to educate users about tracks.

A line (track) with two pins on it may be a good option.

We plan to support custom pin icons in the future. But keeping the direct correlation between a button and symbols on the map is important.

Another question arises: now we use a star icon to save and remove a bookmark. We can leverage a familiar material icon instead:
<img width="156" alt="image" src="https://github.com/organicmaps/organicmaps/assets/170263/7f9cec90-3d31-4465-8853-f662f9c8d501">

**Comment by pastk** [comment]
> A line (track) with two pins on it may be a good option.

I think users will associate it with route building..

**Comment by biodranik** [comment]
Good points. How to better differentiate route building from "stored routes and places"?

**Comment by biodranik** [comment]
I forgot to mention that non-categorized bookmark pins on the map have a star symbol inside them. So there is _some_ connection between the star at the bottom and bookmarks on the map.

Let's investigate how bookmarks are named, displayed on the map, and displayed in the UI/lists in other popular map apps to make it easier for experienced users to migrate, and new users to learn.

**Comment by Jean-BaptisteC** [comment]
- Google Maps uses a bookmark icon to add in bookmarks list and pins on map don't have an icon.
- Waze uses a stars icon to add in bookmarks list and pin on the map don't have an icon
- Apple plans use a plus icon to add in bookmarks list and pin on the map don't have icon
- OsmAnd uses a stars icon to add in bookmarks list and uses stars icon in map pin

A middle solution is to use new bookmark icons in app bottom bar and in place page buttons and keep stars icons in other places.

**Comment by biodranik** [comment]
Apple:
1. A pin icon with a "Dropped Pin" text on the empty map.
2. A small star icon at the top right of the bookmarked POI icon on the map.
3. "Add to Favourites" button with an empty star icon
4. Circle with a star inside for the "Favourites" list menu item.

Note that after refactoring to render SVG icons on map #4772, it will be easy to generate custom bookmark pin icons with a POI icon inside.

Also many users requested completely custom bookmark icons from a png/jpg like it is implemented in Google Earth/KML #2795
